README.md

```diff
@@ -1,39 +1,39 @@
-# Ubuntu 22.04 based MLflow image v2.1.1
-This repository regularly builds new [Mlflow](https://github.com/mlflow/mlflow) image base on LTS Ubuntu 22.04. The process weekly downloads the upstream MLflow v2.1.1 and changes the base for Docker image with the Dockerfile you can see in this repository. After the new image is built the ci runs simple tests against the image and runs the [trivy](https://github.com/aquasecurity/trivy) vulnerability scan. The results are published to the security tab in the repository (only visible to admins). 
 
-The image is currently published [here](https://hub.docker.com/r/charmed/base-mlflow)
-You can use this image seamlesly as any other python Docker image by specifying it as a base with.
+# Ubuntu 22.04 based MLflow image v2.15.0
+This repository regularly builds a new [MLflow](https://github.com/mlflow/mlflow) image based on LTS Ubuntu 22.04. After the new image is built, the CI runs simple tests against the image and runs the [trivy](https://github.com/aquasecurity/trivy) vulnerability scan. 
```

Comment:
I understand that the part about the CI is not entirely true. And also, it's not clear to me when what this happens. I mean if we refer to the PR CI, then this builds, scans, tests and also publishes the image.